amazon-s3/6.2/modules/ROOT/pages/amazon-s3-connector-reference.adoc

```diff
@@ -50,7 +51,8 @@ Uses the S3 connection type, which covers both the Basic and Role connections.
 ** HOURS
 ** DAYS |  Time unit for the *Response Timeout* field. |  `SECONDS` |
 | Role a| <<Role>> |  Role configuration. |  |
-| Region Endpoint a| String |  Topic region endpoint. |  `us-east-1` |
+| Try Default AWSCredentials Provider Chain a| Boolean |  Set this field to `true` to obtain credentials from the AWS environment. For more information, refer to https://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/credentials.html[Provide Temporary Credentials to the SDK
```

Comment:
An optional suggestion to tighten the wording: 

If `true`, obtains credentials from the AWS environment